vault/resource_database_secret_backend_connection.go

```diff
@@ -315,6 +315,39 @@ func connectionStringResource() *schema.Resource {
 	}
 }
 
+func connectionStringResourceWithUsernameTemplate() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"connection_url": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Connection string to use to connect to the database.",
```

Comment:
```suggestion
				Description: "Database connection URL.",
```

Reply:
Updated the description in `connectionStringResourceWithoutUsernameTemplate`